### ISSUE TITLE
Color scales

### ISSUE THREAD

**Issue by nofurtherinformation** [chihacknight/chicago-elections-archive#7]
We should revisit color scales. The maps primarily use continuous unclassified scales currently, which are easy to implement but not always the most helpful.

My default is quantile scales with explicit breakpoints:

https://observablehq.com/@d3/quantile-quantize-and-threshold-scales

**Comment by derekeder** [comment]
agreed, creating our own buckets would be a great idea. also a fan of [Jenks Natural Breaks](https://en.wikipedia.org/wiki/Jenks_natural_breaks_optimization)
